pkg/server/intent.go

```diff
@@ -33,6 +42,14 @@ func (s *Server) ListIntent(ctx context.Context, req *sdcpb.ListIntentRequest) (
 }
 
 func (s *Server) GetIntent(ctx context.Context, req *sdcpb.GetIntentRequest) (*sdcpb.GetIntentResponse, error) {
+	log := logf.FromContext(ctx).WithName("GetIntent")
+	log = log.WithValues(
+		"intent-datastore", req.GetDatastoreName(),
```

Comment:
same here, log once... maybe create a unique id that is to be added for tracing of the request ?!